portfolio/src/main/webapp/script.js

```diff
@@ -79,3 +79,9 @@ function createCommentElement(comment) {
   return commentElement;
 }
 
+function deleteComments() {
+    const request = new Request('/delete-comment', {method: 'POST'});
```

Comment:
Nit: Resource deletions conventionally use the HTTP `DELETE` verb, not `POST`.

Reply:
Of course! I wouldn't write "nit" if I thought my review comment was very important. I invite you to disregard such nitpicks for any reason--and following project instructions is a pretty good one.